lib/cucumber/rake/task.rb

```diff
@@ -113,7 +113,15 @@ def run
       attr_reader :cucumber_opts
 
       def cucumber_opts=(opts) # :nodoc:
-        @cucumber_opts = String == opts.class ? opts.split(' ') : opts
+        unless String == opts.class
+          @cucumber_opts = opts
```

Comment:
can just be return @cucumber_opts = opts

Reply:
I did that first but it triggered a rubocop offense: Lint/ReturnInVoidContext

Reply:
Yeh I think it's because you have a writer method here. Rubocop actually ignores this issue and admits there's a problem. So I'm happy with what you've done for now. Definitely a bit ugly but if the ruby style guide can't agree, there's no reason for us to go off.